release-tools/scripts/manifest.yml

```diff
@@ -0,0 +1,226 @@
+  - 
+    plugin_git: opendistro-for-elasticsearch/kibana-notebooks
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: []
+    plugin_type: [zip]
+    plugin_category: kibana
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/kibana-plugins/notebooks"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/kibana-plugins/opendistro-notebooks"]
+  - 
+    plugin_git: opendistro-for-elasticsearch/sql
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: []
+    plugin_type: [tar.gz,whl]
+    plugin_category: client
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-clients/sql-cli"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/elasticsearch-clients/opendistro-sql-cli", 
+                           "s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/elasticsearch-clients/opendistro-sql-cli"]
+  - 
+    plugin_git: opendistro-for-elasticsearch/sql
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: []
+    plugin_type: [jar]
+    plugin_category: client
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-clients/sql-jdbc"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/elasticsearch-clients/opendistro-sql-jdbc"]
+  - 
+    plugin_git: opendistro-for-elasticsearch/sql
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: [32-bit,64-bit]
+    plugin_type: [msi]
+    plugin_category: client
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-clients/sql-odbc"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/elasticsearch-clients/opendistro-sql-odbc/windows"]
+  - 
+    plugin_git: opendistro-for-elasticsearch/sql
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: []
+    plugin_type: [pkg]
+    plugin_category: client
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-clients/sql-odbc"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/elasticsearch-clients/opendistro-sql-odbc/mac"]
+  - 
+    plugin_git: opendistro-for-elasticsearch/perftop
+    plugin_version: "1.11.0.0"
+    plugin_build: ""
+    plugin_keyword: [MACOS,LINUX]
+    plugin_type: [zip]
+    plugin_category: client
+    plugin_location_staging: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshots/elasticsearch-clients/perftop"
+    plugin_location_prod: ["s3://artifacts.opendistroforelasticsearch.amazon.com/downloads/perftop"]
+
+releases: 
+  url: "s3://staging.artifacts.opendistroforelasticsearch.amazon.com/releases/1.11.0"
+versions:
```

Comment:
Please move this to the top since this is the most important thing in the release and other things are dependent on it.